src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+/**
+ * Deadline task
+ */
+class Deadline extends Task implements DateTimeInterface {
+    private final LocalDate date;
+
+    public Deadline(String task, LocalDate date) {
+        super(task, "D");
+        this.date = date;
+    }
+
+    public Deadline(String task, boolean complete, LocalDate date) {
```

Comment:
Should the variable complete have a prefix (isComplete, hasCompleted) since it is a boolean variable?